game.py

```diff
@@ -20,7 +20,55 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    # pass
+    # 1. User starts the game from the command line
+    # 2. A secret word is selected
```

Comment:
Great use of pseudocode to plan out your approach! When we get ready to submit code for review here at Ada, and especially once we're on a development team, an important step of preparing our code for submission is to remove planning comments, debug statements, and anything else that isn't necessary to making our code function and be easy to understand. 

It's great to leave a few short comments through the code to help explain long or complicated sections, but comments can make it hard to see the code all together, and also need to be maintained over time to ensure they do not fall out of date with the code they surround. A conflicting or confusing comment in code is often less helpful than no comment at all.